keyboards/sx60/readme.md

```diff
@@ -0,0 +1,18 @@
+SX60
+===
+
+![SX60](https://i.imgur.com/hZZHrRr.jpg)
+
+Keyboard Maintainer: [amnobis](https://github.com/amnobis)
```

Comment:
Could you make sure that these three lines have double spaces at the end of the lines, so that they are parsed correctly?

Otherwise, they'll end up on one line.